src/opm/input/eclipse/EclipseState/Grid/Carfin.cpp

```diff
@@ -75,6 +75,15 @@ namespace {
 
 namespace Opm
 {
+
+    Carfin::Carfin(const std::string& lgrName) :
+        m_dims({2,2,2}),   
+        m_offset({1,1,1}),   
+        m_end_offset({2,2,2}),  
```

Comment:
How did you select these values?  Are they reasonable defaults for all `Carfin` objects?